src/commands/Notifications/youtube-subscription.ts

```diff
@@ -0,0 +1,162 @@
+import { envParseBoolean } from '#lib/env';
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import type { GuildMessage } from '#lib/types';
+import { PermissionLevels } from '#lib/types/Enums';
+import { hyperlink } from '@discordjs/builders';
+import { ApplyOptions, RequiresClientPermissions } from '@sapphire/decorators';
+import { Args, CommandOptionsRunTypeEnum, Resolvers } from '@sapphire/framework';
+import { send } from '@sapphire/plugin-editable-commands';
+import { PermissionFlagsBits } from 'discord-api-types/v9';
+import { MessageEmbed } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	enabled: envParseBoolean('GRPC_NOTIFICATIONS_ENABLED'),
+	aliases: ['y-subscription', 'y-sub'],
+	description: LanguageKeys.Commands.Notifications.YouTubeSubscriptionDescription,
+	detailedDescription: LanguageKeys.Commands.Notifications.YouTubeSubscriptionExtended,
+	permissionLevel: PermissionLevels.Administrator,
+	runIn: [CommandOptionsRunTypeEnum.GuildAny],
+	subCommands: [
+		'set',
+		'add',
+		{ input: 'subscribe', output: 'add' },
+		'remove',
+		{ input: 'unsubscribe', output: 'remove' },
+		'clear',
+		{ input: 'list', default: true }
+	]
+})
+export class UserCommand extends SkyraCommand {
+	public async add(message: GuildMessage, args: SkyraCommand.Args) {
+		const channelUrl = await args.pick(UserCommand.youtubeUrl);
+		await this.container.grpc.youtube.subscribe({ guildId: message.guild.id, channelUrl });
+		return send(message, args.t(LanguageKeys.Commands.Notifications.YouTubeSubscriptionAdd, { channelUrl }));
+	}
+
+	public async remove(message: GuildMessage, args: SkyraCommand.Args) {
+		const channelUrl = await args.pick(UserCommand.youtubeUrl);
+		await this.container.grpc.youtube.unsubscribe({ guildId: message.guild.id, channelUrl });
+		return send(message, args.t(LanguageKeys.Commands.Notifications.YouTubeSubscriptionRemove, { channelUrl }));
+	}
+
+	public async clear(message: GuildMessage, args: SkyraCommand.Args) {
+		await this.container.grpc.youtube.removeAllSubscriptions({ guildId: message.guild.id });
+		return send(message, args.t(LanguageKeys.Commands.Notifications.YouTubeSubscriptionClear));
+	}
+
+	@RequiresClientPermissions(PermissionFlagsBits.EmbedLinks)
+	public async list(message: GuildMessage, args: SkyraCommand.Args) {
+		const { infoList } = await this.container.grpc.youtube.getSubscriptions({ guildId: message.guild.id });
+		if (infoList.length === 0) {
+			this.error(LanguageKeys.Commands.Notifications.YouTubeSubscriptionListEmpty);
+		}
+
+		const entries = infoList.map((entry) => hyperlink(entry.channelName, entry.channelUrl));
```

Comment:
Do we want to also use `hideLinkEmbed` here just in case? I know it doesn't embed it because its send in an embed but still... 

lmk what you think.

Reply:
Nah, not needed.